content/en/docs/guides/keepalive.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Keepalive
+description: >-
+  How to use HTTP/2 PING-based keepalives in gRPC
+---
+
+### Overview
+
+HTTP/2 PING-based keepalives are a way to keep a HTTP/2 connection alive even when there is no data being transferred. This is done by periodically sending a PING frame to the other end of the connection. HTTP/2 keepalives can improve performance and reliability of HTTP/2 connections, but it is important to configure the keepalive interval carefully.
+
+{{% alert title="Note" color="info" %}}
+  There is a related but separate concern called [Health Checking]. Health checking allows server to signal whether a *service* is healthy while keepalive is only about the *connection*.
```

Comment:
Not sure they are related and if anyone will confuse between the 2. Just my opinion